Lib/subprocess.py

```diff
@@ -1307,15 +1307,15 @@ def _execute_child(self, args, executable, preexec_fn, close_fds,
                 try:
```

Comment:
Have you read my comment here? https://github.com/python/cpython/pull/3066#discussion_r133686665

In the case where _posixsubprocess writes to the errpipe, there should be nothing in there that can't be decoded. And if there is junk in there that wasn't written by _posixsubproccess, the decode will fail and it'll print out the appropriate message saying `Bad exception data from child: b'UNDECODABLE:MESS:HERE'`